packages/um/package.py

```diff
@@ -304,6 +324,28 @@ def _build_dir(self):
         return join_path(self.stage.source_path, "..", "spack-build")
 
 
+    def patch(self):
+        """Patch the staging directory just before building."""
+        
+        # Get the root to the resources
+        resources_root = join_path(self.stage.source_path, "resources")
```

Comment:
Consider defining the string "resources" as a private member variable of the class.